### PR TITLE
Mark eligible recipes as primary

### PR DESCRIPTION
**Pull request by michaeljguarino**
## Summary
will be useful for distinguishing on catalog pages and cli commands

## Test Plan
n/a

## Checklist

- [ ]  No images hosted from dockerhub
- [ ]  Are dashboards present to understand the health of the application.  There **must** be at least 1 of these
    - [ ]  all databases should have dashboards
    - [ ]  ideally also have at least cpu/mem utilization dashboards for webserver tier of the app
    - [ ]  you can use `plural from-grafana` to convert a grafana dashboard found via google to our CRD
- [ ]  Are scaling runbooks present
    - [ ]  all databases **must** have scaling runbooks
    - [ ]  you can use the charts in `pluralsh/module-library` to accelerate this
- [ ]  do you need to add config overlays?
    - [ ]  inputing secrets
    - [ ]  configuring autoscaling
- [ ]  If there’s a web-facing component to the app, we need to support OIDC authentication and setting up private networks if no authentication option is viable
- [ ]  All major clouds must be supported
    - [ ]  Azure
    - [ ]  AWS
    - [ ]  GCP

### CODE REVIEW

File: hydra/plural/recipes/hydra-azure.yaml [hunk 1]
```diff
@@ -15,7 +16,8 @@ sections:
     documentation: domain on which you'd like to host hydra
     type: DOMAIN
   - name: adminHostname
-    documentation: dns name for the internal endpoint for hydra admin (leave empty to not enable ingress)
+    documentation: dns name for the internal endpoint for hydra admin (leave empty
```
Comment (DavidSpek) [inline hydra/plural/recipes/hydra-azure.yaml:19]: Same here, probably shouldn't be 2 lines.

File: hydra/plural/recipes/hydra-gcp.yaml [hunk 2]
```diff
@@ -15,7 +16,8 @@ sections:
     documentation: domain on which you'd like to host hydra
     type: DOMAIN
   - name: adminHostname
-    documentation: dns name for the internal endpoint for hydra admin (leave empty to not enable ingress)
+    documentation: dns name for the internal endpoint for hydra admin (leave empty
+      to not enable ingress)
```
Comment (DavidSpek) [inline hydra/plural/recipes/hydra-gcp.yaml:20]: Same here, probably shouldn't be 2 lines

File: influx/plural/recipes/influx-aws.yaml [hunk 3]
```diff
@@ -29,7 +30,8 @@ sections:
     documentation: name for the initial bootstrapped database
     type: STRING
   - name: influxdbHostname
-    documentation: external dns name for your influxdb instance (leave empty if you don't want ingress)
+    documentation: external dns name for your influxdb instance (leave empty if you
+      don't want ingress)
```
Comment (DavidSpek) [inline influx/plural/recipes/influx-aws.yaml:34]: Same here, probably shouldn't be 2 lines

File: influx/plural/recipes/influx-azure.yaml [hunk 4]
```diff
@@ -29,7 +30,8 @@ sections:
     documentation: name for the initial bootstrapped database
     type: STRING
   - name: influxdbHostname
-    documentation: external dns name for your influxdb instance (leave empty if you don't want ingress)
+    documentation: external dns name for your influxdb instance (leave empty if you
+      don't want ingress)
```
Comment (DavidSpek) [inline influx/plural/recipes/influx-azure.yaml:34]: Same here, probably shouldn't be 2 lines

File: influx/plural/recipes/influx-gcp.yaml [hunk 5]
```diff
@@ -29,7 +30,8 @@ sections:
     documentation: name for the initial bootstrapped database
     type: STRING
   - name: influxdbHostname
-    documentation: external dns name for your influxdb instance (leave empty if you don't want ingress)
+    documentation: external dns name for your influxdb instance (leave empty if you
+      don't want ingress)
```
Comment (DavidSpek) [inline influx/plural/recipes/influx-gcp.yaml:34]: Same here, probably shouldn't be 2 lines

File: kratos/plural/recipes/kratos-aws.yaml [hunk 6]
```diff
@@ -15,11 +16,13 @@ sections:
     documentation: domain on which you'd like to host hydra
     type: DOMAIN
   - name: adminHostname
-    documentation: dns name for the internal endpoint for hydra admin (leave empty to not enable ingress)
+    documentation: dns name for the internal endpoint for hydra admin (leave empty
+      to not enable ingress)
```
Comment (DavidSpek) [inline kratos/plural/recipes/kratos-aws.yaml:20]: Same here, probably shouldn't be 2 lines

File: kratos/plural/recipes/kratos-aws.yaml [hunk 7]
```diff
@@ -15,11 +16,13 @@ sections:
...
     type: DOMAIN
   - name: returnUrl
     type: STRING
-    documentation: the default url kratos will redirect to on completion of login flows
+    documentation: the default url kratos will redirect to on completion of login
+      flows
```
Comment (DavidSpek) [inline kratos/plural/recipes/kratos-aws.yaml:25]: Same here, probably shouldn't be 2 lines

File: kratos/plural/recipes/kratos-azure.yaml [hunk 8]
```diff
@@ -15,11 +16,13 @@ sections:
     documentation: domain on which you'd like to host hydra
     type: DOMAIN
   - name: adminHostname
-    documentation: dns name for the internal endpoint for hydra admin (leave empty to not enable ingress)
+    documentation: dns name for the internal endpoint for hydra admin (leave empty
+      to not enable ingress)
```
Comment (DavidSpek) [inline kratos/plural/recipes/kratos-azure.yaml:20]: Same here, probably shouldn't be 2 lines

File: kratos/plural/recipes/kratos-azure.yaml [hunk 9]
```diff
@@ -15,11 +16,13 @@ sections:
...
     type: DOMAIN
   - name: returnUrl
     type: STRING
-    documentation: the default url kratos will redirect to on completion of login flows
+    documentation: the default url kratos will redirect to on completion of login
+      flows
```
Comment (DavidSpek) [inline kratos/plural/recipes/kratos-azure.yaml:25]: Same here, probably shouldn't be 2 lines

File: kratos/plural/recipes/kratos-gcp.yaml [hunk 10]
```diff
@@ -15,11 +16,13 @@ sections:
     documentation: domain on which you'd like to host hydra
     type: DOMAIN
   - name: adminHostname
-    documentation: dns name for the internal endpoint for hydra admin (leave empty to not enable ingress)
+    documentation: dns name for the internal endpoint for hydra admin (leave empty
+      to not enable ingress)
```
Comment (DavidSpek) [inline kratos/plural/recipes/kratos-gcp.yaml:20]: Same here, probably shouldn't be 2 lines

File: minio/plural/recipes/minio-aws.yaml [hunk 11]
```diff
@@ -16,10 +17,14 @@ sections:
...
     documentation: bucket to store minio data
   - name: hostname
-    documentation: Fully Qualified Domain Name to use for your minio gateway installation, eg minio.topleveldomain.com if topleveldomain.com is the domain you inputed for dns_domain above.
+    documentation: Fully Qualified Domain Name to use for your minio gateway installation,
+      eg minio.topleveldomain.com if topleveldomain.com is the domain you inputed
+      for dns_domain above.
     type: DOMAIN
   - name: consoleHostname
-    documentation: Fully Qualified Domain Name to use for your minio console installation, eg minio.topleveldomain.com if topleveldomain.com is the domain you inputed for dns_domain above.
+    documentation: Fully Qualified Domain Name to use for your minio console installation,
```
Comment (DavidSpek) [inline minio/plural/recipes/minio-aws.yaml:25]: Same here, probably shouldn't be 2 lines

File: minio/plural/recipes/minio-azure.yaml [hunk 12]
```diff
@@ -12,10 +13,13 @@ sections:
 - name: minio
   configuration:
   - name: hostname
-    documentation: Fully Qualified Domain Name to use for your minio gateway installation, eg minio.topleveldomain.com if topleveldomain.com is the domain you inputed for dns_domain above.
+    documentation: Fully Qualified Domain Name to use for your minio gateway installation,
```
Comment (DavidSpek) [inline minio/plural/recipes/minio-azure.yaml:16]: Same here, probably shouldn't be 2 lines

File: minio/plural/recipes/minio-azure.yaml [hunk 13]
```diff
@@ -12,10 +13,13 @@ sections:
...
+      eg minio.topleveldomain.com if topleveldomain.com is the domain you inputed
+      for dns_domain above.
     type: DOMAIN
   - name: consoleHostname
-    documentation: domain name for the minio console.  This MUST be different from the domain for minio itself, configured earlier
+    documentation: domain name for the minio console.  This MUST be different from
```
Comment (DavidSpek) [inline minio/plural/recipes/minio-azure.yaml:21]: Same here, probably shouldn't be 2 lines

File: minio/plural/recipes/minio-equinix.yaml [hunk 14]
```diff
@@ -14,10 +15,14 @@ sections:
 - name: minio
   configuration:
   - name: hostname
-    documentation: Fully Qualified Domain Name to use for your minio gateway installation, eg minio.topleveldomain.com if topleveldomain.com is the domain you inputed for dns_domain above.
+    documentation: Fully Qualified Domain Name to use for your minio gateway installation,
```
Comment (DavidSpek) [inline minio/plural/recipes/minio-equinix.yaml:18]: Same here, probably shouldn't be 2 lines

File: minio/plural/recipes/minio-equinix.yaml [hunk 15]
```diff
@@ -14,10 +15,14 @@ sections:
...
+      eg minio.topleveldomain.com if topleveldomain.com is the domain you inputed
+      for dns_domain above.
     type: DOMAIN
   - name: consoleHostname
-    documentation: Fully Qualified Domain Name to use for your minio console installation, eg minio.topleveldomain.com if topleveldomain.com is the domain you inputed for dns_domain above.
+    documentation: Fully Qualified Domain Name to use for your minio console installation,
```
Comment (DavidSpek) [inline minio/plural/recipes/minio-equinix.yaml:23]: Same here, probably shouldn't be 2 lines

File: minio/plural/recipes/minio-gcp.yaml [hunk 16]
```diff
@@ -12,10 +13,14 @@ sections:
 - name: minio
   configuration:
   - name: hostname
-    documentation: Fully Qualified Domain Name to use for your minio gateway installation, eg minio.topleveldomain.com if topleveldomain.com is the domain you inputed for dns_domain above.
+    documentation: Fully Qualified Domain Name to use for your minio gateway installation,
```
Comment (DavidSpek) [inline minio/plural/recipes/minio-gcp.yaml:16]: Same here, probably shouldn't be 2 lines

File: minio/plural/recipes/minio-gcp.yaml [hunk 17]
```diff
@@ -12,10 +13,14 @@ sections:
...
+      eg minio.topleveldomain.com if topleveldomain.com is the domain you inputed
+      for dns_domain above.
     type: DOMAIN
   - name: consoleHostname
-    documentation: Fully Qualified Domain Name to use for your minio console installation, eg minio.topleveldomain.com if topleveldomain.com is the domain you inputed for dns_domain above.
+    documentation: Fully Qualified Domain Name to use for your minio console installation,
```
Comment (DavidSpek) [inline minio/plural/recipes/minio-gcp.yaml:21]: Same here, probably shouldn't be 2 lines

File: minio/plural/recipes/minio-kind.yaml [hunk 18]
```diff
@@ -12,10 +13,14 @@ sections:
 - name: minio
   configuration:
   - name: hostname
-    documentation: Fully Qualified Domain Name to use for your minio gateway installation, eg minio.topleveldomain.com if topleveldomain.com is the domain you inputed for dns_domain above.
+    documentation: Fully Qualified Domain Name to use for your minio gateway installation,
```
Comment (DavidSpek) [inline minio/plural/recipes/minio-kind.yaml:16]: Same here, probably shouldn't be 2 lines

File: postgres/plural/recipes/gcp-postgres.yaml [hunk 19]
```diff
@@ -1,14 +1,16 @@
...
-  - repo: bootstrap
-    name: gcp-k8s
+- repo: bootstrap
+  name: gcp-k8s
 sections:
 - name: postgres
   configuration:
   - name: wal_bucket
-    documentation: Arbitrary name for google cloud storage bucket to store wal archives in, eg plural-wal-archives
+    documentation: Arbitrary name for google cloud storage bucket to store wal archives
```
Comment (DavidSpek) [inline postgres/plural/recipes/gcp-postgres.yaml:12]: Same here, probably shouldn't be 2 lines

File: renovate-on-prem/plural/recipes/renovate-on-prem-aws.yaml [hunk 20]
```diff
@@ -24,14 +25,16 @@ sections:
     default: github
     type: STRING
   - name: githubAppId
-    documentation: The GitHub App ID provided by GitHub Enterprise when you provisioned the Renovate app.
+    documentation: The GitHub App ID provided by GitHub Enterprise when you provisioned
```
Comment (DavidSpek) [inline renovate-on-prem/plural/recipes/renovate-on-prem-aws.yaml:28]: Same here, probably shouldn't be 2 lines

File: renovate-on-prem/plural/recipes/renovate-on-prem-aws.yaml [hunk 21]
```diff
@@ -24,14 +25,16 @@ sections:
...
+      the Renovate app.
     type: STRING
     condition:
       operation: EQ
       field: platform
       value: github
   - name: githubAppKey
-    documentation: A string representation of the private key provided by GitHub when you provisioned the Renovate app.
+    documentation: A string representation of the private key provided by GitHub when
```
Comment (DavidSpek) [inline renovate-on-prem/plural/recipes/renovate-on-prem-aws.yaml:36]: Same here, probably shouldn't be 2 lines

File: renovate-on-prem/plural/recipes/renovate-on-prem-azure.yaml [hunk 22]
```diff
@@ -24,14 +25,16 @@ sections:
...
+    documentation: The GitHub App ID provided by GitHub Enterprise when you provisioned
+      the Renovate app.
     type: STRING
     condition:
       operation: EQ
       field: platform
       value: github
   - name: githubAppKey
-    documentation: A string representation of the private key provided by GitHub when you provisioned the Renovate app.
+    documentation: A string representation of the private key provided by GitHub when
```
Comment (DavidSpek) [inline renovate-on-prem/plural/recipes/renovate-on-prem-azure.yaml:36]: Same here, probably shouldn't be 2 lines

File: renovate-on-prem/plural/recipes/renovate-on-prem-azure.yaml [hunk 23]
```diff
@@ -49,7 +52,9 @@ sections:
       field: platform
       value: gitlab
   - name: githubComToken
-    documentation: A Personal Access Token for a valid user account on github.com. This is only used for retrieving changelogs and release notes from repositories hosted on github.com.
+    documentation: A Personal Access Token for a valid user account on github.com.
```
Comment (DavidSpek) [inline renovate-on-prem/plural/recipes/renovate-on-prem-azure.yaml:55]: Same here, probably shouldn't be 2 lines

File: renovate-on-prem/plural/recipes/renovate-on-prem-gcp.yaml [hunk 24]
```diff
@@ -24,14 +25,16 @@ sections:
...
+    documentation: The GitHub App ID provided by GitHub Enterprise when you provisioned
+      the Renovate app.
     type: STRING
     condition:
       operation: EQ
       field: platform
       value: github
   - name: githubAppKey
-    documentation: A string representation of the private key provided by GitHub when you provisioned the Renovate app.
+    documentation: A string representation of the private key provided by GitHub when
```
Comment (DavidSpek) [inline renovate-on-prem/plural/recipes/renovate-on-prem-gcp.yaml:36]: Same here, probably shouldn't be 2 lines

File: retool/plural/recipes/retool-azure.yaml [hunk 25]
```diff
@@ -16,7 +17,8 @@ sections:
     documentation: the domain name you want retool to be accessible from
   - name: licenseKey
     type: PASSWORD
-    documentation: your retool license key, accessible in the retool dashboard under self-hosted
+    documentation: your retool license key, accessible in the retool dashboard under
```
Comment (DavidSpek) [inline retool/plural/recipes/retool-azure.yaml:20]: Same here, probably shouldn't be 2 lines

File: sftpgo/plural/recipes/sftpgo-azure.yaml [hunk 26]
```diff
@@ -1,19 +1,21 @@
...
-      - type: TERRAFORM
-        name: kube
-      - type: HELM
-        name: sftpgo
+- name: sftpgo
+  configuration:
+  - name: hostname
+    type: DOMAIN
+    documentation: the fully qualified domain name your sftpgo instance will be available
+      at
```
Comment (DavidSpek) [inline sftpgo/plural/recipes/sftpgo-azure.yaml:16]: Same here, probably shouldn't be 2 lines

File: sftpgo/plural/recipes/sftpgo-gcp.yaml [hunk 27]
```diff
@@ -1,19 +1,21 @@
...
-      - type: TERRAFORM
-        name: kube
-      - type: HELM
-        name: sftpgo
+- name: sftpgo
+  configuration:
+  - name: hostname
+    type: DOMAIN
+    documentation: the fully qualified domain name your sftpgo instance will be available
+      at
```
Comment (DavidSpek) [inline sftpgo/plural/recipes/sftpgo-gcp.yaml:16]: Same here, probably shouldn't be 2 lines

File: sftpgo/plural/recipes/sftpgo-kind.yaml [hunk 28]
```diff
@@ -1,19 +1,21 @@
...
-      - type: TERRAFORM
-        name: kube
-      - type: HELM
-        name: sftpgo
+- name: sftpgo
+  configuration:
+  - name: hostname
+    type: DOMAIN
+    documentation: the fully qualified domain name your sftpgo instance will be available
+      at
```
Comment (DavidSpek) [inline sftpgo/plural/recipes/sftpgo-kind.yaml:16]: Same here, probably shouldn't be 2 lines

File: superset/plural/recipes/superset-aws.yaml [hunk 29]
```diff
@@ -16,7 +17,9 @@ sections:
 - name: superset
   configuration:
   - name: hostname
-    documentation: Fully Qualified Domain Name to use for your superset installation, eg airflow.topleveldomain.com if topleveldomain.com is the domain you inputed for dns_domain above.
+    documentation: Fully Qualified Domain Name to use for your superset installation,
```
Comment (DavidSpek) [inline superset/plural/recipes/superset-aws.yaml:20]: Same here, probably shouldn't be 2 lines

File: superset/plural/recipes/superset-azure.yaml [hunk 30]
```diff
@@ -1,22 +1,25 @@
...
+- repo: ingress-nginx
+  name: ingress-nginx-azure
+- repo: postgres
+  name: azure-postgres
 sections:
 - name: superset
   configuration:
   - name: hostname
-    documentation: Fully Qualified Domain Name to use for your superset installation, eg airflow.topleveldomain.com if topleveldomain.com is the domain you inputed for dns_domain above.
+    documentation: Fully Qualified Domain Name to use for your superset installation,
```
Comment (DavidSpek) [inline superset/plural/recipes/superset-azure.yaml:20]: Same here, probably shouldn't be 2 lines

File: superset/plural/recipes/superset-gcp.yaml [hunk 31]
```diff
@@ -1,22 +1,25 @@
...
+- repo: ingress-nginx
+  name: ingress-nginx-gcp
+- repo: postgres
+  name: gcp-postgres
 sections:
 - name: superset
   configuration:
   - name: hostname
-    documentation: Fully Qualified Domain Name to use for your superset installation, eg airflow.topleveldomain.com if topleveldomain.com is the domain you inputed for dns_domain above.
+    documentation: Fully Qualified Domain Name to use for your superset installation,
```
Comment (DavidSpek) [inline superset/plural/recipes/superset-gcp.yaml:20]: Same here, probably shouldn't be 2 lines

File: terraria/plural/recipes/terraria-aws.yaml [hunk 32]
```diff
@@ -15,16 +16,18 @@ sections:
     type: STRING
     default: small
   - name: password
-    documentation: password that will be required when joining the server (leave empty to disable)
+    documentation: password that will be required when joining the server (leave empty
```
Comment (DavidSpek) [inline terraria/plural/recipes/terraria-aws.yaml:19]: Same here, probably shouldn't be 2 lines

File: terraria/plural/recipes/terraria-aws.yaml [hunk 33]
```diff
@@ -15,16 +16,18 @@ sections:
...
+      to disable)
     type: STRING
     optional: true
     default: ''
   - name: restAPIEnabled
-    documentation: if additional rest API should be enabled and exposed on port 7878 (Y/n)
+    documentation: if additional rest API should be enabled and exposed on port 7878
```
Comment (DavidSpek) [inline terraria/plural/recipes/terraria-aws.yaml:25]: Same here, probably shouldn't be 2 lines

File: terraria/plural/recipes/terraria-azure.yaml [hunk 34]
```diff
@@ -15,16 +16,18 @@ sections:
     type: STRING
     default: small
   - name: password
-    documentation: password that will be required when joining the server (leave empty to disable)
+    documentation: password that will be required when joining the server (leave empty
```
Comment (DavidSpek) [inline terraria/plural/recipes/terraria-azure.yaml:19]: Same here, probably shouldn't be 2 lines

File: terraria/plural/recipes/terraria-azure.yaml [hunk 35]
```diff
@@ -15,16 +16,18 @@ sections:
...
+      to disable)
     type: STRING
     optional: true
     default: ''
   - name: restAPIEnabled
-    documentation: if additional rest API should be enabled and exposed on port 7878 (Y/n)
+    documentation: if additional rest API should be enabled and exposed on port 7878
```
Comment (DavidSpek) [inline terraria/plural/recipes/terraria-azure.yaml:25]: Same here, probably shouldn't be 2 lines

File: terraria/plural/recipes/terraria-equinix.yaml [hunk 36]
```diff
@@ -15,16 +16,18 @@ sections:
     type: STRING
     default: small
   - name: password
-    documentation: password that will be required when joining the server (leave empty to disable)
+    documentation: password that will be required when joining the server (leave empty
```
Comment (DavidSpek) [inline terraria/plural/recipes/terraria-equinix.yaml:19]: Same here, probably shouldn't be 2 lines

File: terraria/plural/recipes/terraria-gcp.yaml [hunk 37]
```diff
@@ -15,16 +16,18 @@ sections:
     type: STRING
     default: small
   - name: password
-    documentation: password that will be required when joining the server (leave empty to disable)
+    documentation: password that will be required when joining the server (leave empty
```
Comment (DavidSpek) [inline terraria/plural/recipes/terraria-gcp.yaml:19]: Same here, probably shouldn't be 2 lines

File: terraria/plural/recipes/terraria-kind.yaml [hunk 38]
```diff
@@ -15,16 +16,18 @@ sections:
...
   - name: password
-    documentation: password that will be required when joining the server (leave empty to disable)
+    documentation: password that will be required when joining the server (leave empty
+      to disable)
     type: STRING
     optional: true
     default: ''
   - name: restAPIEnabled
-    documentation: if additional rest API should be enabled and exposed on port 7878 (Y/n)
+    documentation: if additional rest API should be enabled and exposed on port 7878
```
Comment (DavidSpek) [inline terraria/plural/recipes/terraria-kind.yaml:25]: Same here, probably shouldn't be 2 lines

File: touca/plural/recipes/touca-aws.yaml [hunk 39]
```diff
@@ -13,7 +14,8 @@ sections:
   configuration:
   - name: hostname
     type: DOMAIN
-    documentation: the fully qualified domain name your touca instance will be available at
+    documentation: the fully qualified domain name your touca instance will be available
```
Comment (DavidSpek) [inline touca/plural/recipes/touca-aws.yaml:17]: Same here, probably shouldn't be 2 lines

File: valheim/plural/recipes/valheim-aws.yaml [hunk 40]
```diff
@@ -1,40 +1,42 @@
...
+  - name: worldName
+    documentation: name of the world (without file extension)
+    type: STRING
+    default: Dedicated
+  - name: hostname
+    documentation: domain on which you would like to host your valheim server
+    type: DOMAIN
+  - name: password
+    documentation: password that will be required when joining the server. Minimum
+      5 characters (leave empty to disable).
```
Comment (DavidSpek) [inline valheim/plural/recipes/valheim-aws.yaml:24]: Same here, probably shouldn't be 2 lines

File: valheim/plural/recipes/valheim-azure.yaml [hunk 41]
```diff
@@ -1,41 +1,42 @@
...
+  - name: worldName
+    documentation: name of the world (without file extension)
+    type: STRING
+    default: Dedicated
+  - name: hostname
+    documentation: domain on which you would like to host your valheim server
+    type: DOMAIN
+  - name: password
+    documentation: password that will be required when joining the server. Minimum
+      5 characters (leave empty to disable).
```
Comment (DavidSpek) [inline valheim/plural/recipes/valheim-azure.yaml:24]: Same here, probably shouldn't be 2 lines

File: valheim/plural/recipes/valheim-equinix.yaml [hunk 42]
```diff
@@ -1,41 +1,42 @@
...
+  - name: worldName
+    documentation: name of the world (without file extension)
+    type: STRING
+    default: Dedicated
+  - name: hostname
+    documentation: domain on which you would like to host your valheim server
+    type: DOMAIN
+  - name: password
+    documentation: password that will be required when joining the server. Minimum
+      5 characters (leave empty to disable).
```
Comment (DavidSpek) [inline valheim/plural/recipes/valheim-equinix.yaml:24]: Same here, probably shouldn't be 2 lines

File: valheim/plural/recipes/valheim-gcp.yaml [hunk 43]
```diff
@@ -1,41 +1,42 @@
...
+  - name: worldName
+    documentation: name of the world (without file extension)
+    type: STRING
+    default: Dedicated
+  - name: hostname
+    documentation: domain on which you would like to host your valheim server
+    type: DOMAIN
+  - name: password
+    documentation: password that will be required when joining the server. Minimum
+      5 characters (leave empty to disable).
```
Comment (DavidSpek) [inline valheim/plural/recipes/valheim-gcp.yaml:24]: Same here, probably shouldn't be 2 lines

File: valheim/plural/recipes/valheim-kind.yaml [hunk 44]
```diff
@@ -1,41 +1,42 @@
...
+  - name: worldName
+    documentation: name of the world (without file extension)
+    type: STRING
+    default: Dedicated
+  - name: hostname
+    documentation: domain on which you would like to host your valheim server
+    type: DOMAIN
+  - name: password
+    documentation: password that will be required when joining the server. Minimum
+      5 characters (leave empty to disable).
```
Comment (DavidSpek) [inline valheim/plural/recipes/valheim-kind.yaml:24]: Same here, probably shouldn't be 2 lines

File: yugabyte/plural/recipes/yugabyte-azure.yaml [hunk 45]
```diff
@@ -11,7 +12,8 @@ sections:
   configuration:
   - name: hostname
     type: DOMAIN
-    documentation: the fully qualified domain name your yugabyte instance will be available at
+    documentation: the fully qualified domain name your yugabyte instance will be
```
Comment (DavidSpek) [inline yugabyte/plural/recipes/yugabyte-azure.yaml:15]: Same here, probably shouldn't be 2 lines

File: redis/plural/recipes/aws-redis-cluster.yaml [hunk 46]
```diff
@@ -1,6 +1,7 @@
 name: aws-redis
```
Comment (DavidSpek) [inline redis/plural/recipes/aws-redis-cluster.yaml:1]: should be renamed to `aws-redis-cluster`

File: redis/plural/recipes/azure-redis-cluster.yaml [hunk 47]
```diff
@@ -1,6 +1,7 @@
 name: azure-redis
```
Comment (DavidSpek) [inline redis/plural/recipes/azure-redis-cluster.yaml:1]: Should be renamed to `azure-redis-cluster`.

File: redis/plural/recipes/azure-redis-cluster.yaml [hunk 48]
```diff
@@ -1,6 +1,7 @@
...
 description: Installs redis on an azure aks cluster
 provider: AZURE
+primary: true
```
Comment (DavidSpek) [inline redis/plural/recipes/azure-redis-cluster.yaml:4]: Should be false.

File: redis/plural/recipes/gcp-redis-cluster.yaml [hunk 49]
```diff
@@ -1,6 +1,7 @@
 name: gcp-redis
 description: Installs redis on a GKE cluster
 provider: GCP
+primary: true
```
Comment (DavidSpek) [inline redis/plural/recipes/gcp-redis-cluster.yaml:4]: Should be false.

**Review by DavidSpek — CHANGES_REQUESTED** [review]
Seems like lots of doc strings are being split to 2 lines which will probably break things. Also, there are some fixes needed for the Redis recipes.

**Comment by michaeljguarino** [comment]
The two line strings were the results of python's yaml dump so I'd assume its properly formatted, I'll fix the redis stuff

**Review by DavidSpek — APPROVED** [review]
LGTM

**Pull request merged**